portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -25,7 +25,7 @@
 /**
  * Handles upload request and gets image url.
  */
-@WebServlet("/blobstore-upload-url")
```

Comment:
How is this related to the map api?

Reply:
It's not, but for a while when I had a bug with my blobstore, this was the problem causing it so when i changed it, it messed up with all my PRs as I didn't change it in its specific branch. So this appears in all my PRs now :(